tools/pepsi/src/aliveness.rs

```diff
@@ -0,0 +1,166 @@
+use std::{collections::BTreeMap, net::IpAddr, time::Duration};
+
+use clap::{arg, Args};
+use color_eyre::Result;
+use colored::Colorize;
+
+use crate::parsers::NaoAddress;
+use aliveness::{
+    query_aliveness,
+    service_manager::{ServiceState, SystemServices},
+    AlivenessState,
+};
+use repository::SDK_VERSION;
+
+#[derive(Args)]
+pub struct Arguments {
+    /// Output verbose version of the aliveness information
+    #[arg(long, short = 'v')]
+    verbose: bool,
+    /// Output aliveness information as json
+    #[arg(long, short = 'j')]
+    json: bool,
+    /// Timeout in ms for waiting for responses
+    #[arg(long, short = 't', default_value = "200")]
+    timeout: u64,
```

Comment:
Does clap support `Duration`? This would be type-safe in our code.

Reply:
Not directly, only using either an external crate or by specifying a parser. I now implemented the latter one.